runtime/decimal.c

```diff
@@ -45,21 +45,57 @@ TaggedPtrPanicCode _bal_decimal_sub(TaggedPtr tp1, TaggedPtr tp2) {
     return finish(&d, &cx);    
 }
 
+TaggedPtrPanicCode _bal_decimal_mul(TaggedPtr tp1, TaggedPtr tp2) {
+    const decQuad *dq1 = taggedToDecQuad(tp1);
```

Comment:
Let's inline the calls to taggedToDecQuad in all four functions.

Reply:
Changed